src/main.cpp

```diff
@@ -2,36 +2,56 @@
 
 #include "acceleration.hpp"
 #include "barometer.hpp"
+#include "devices.hpp"
 #include "gps.hpp"
+#include "logger.hpp"
 
 constexpr auto baud_rate = 9600l;
 constexpr int interval = 2000;
-HardwareSerial Serial3{PC11, PC10};
+bool date_append = false;
```

Comment:
bool csv_append_date = true;

Reply:
Jak już zmieniamy nazwę to sugeruje dodać prefix is_